### ISSUE TITLE
Mongo stores floats with 6 decimals and rounds value= 204.8904554713363 to value =204.890455 

### ISSUE THREAD

**Issue by RihabFekii** [Fiware/context.Orion-LD#927]
Better to use Decimal type 
Note:
Unlike the double data type, which only stores an approximation of the decimal values, the decimal data type stores the exact value. For example, a decimal NumberDecimal("9.99") has a precise value of 9.99 where as a double 9.99 would have an approximate value of 9.9900000000000002131628....

Source: https://docs.mongodb.com/v5.0/release-notes/3.4/#decimal-type

**Comment by kzangeli** [comment]
Yeah, the problem is that NumberDecimal seems to be new and is not supported by the older Legacy C++ mongo driver.
One rid of the old driver, let's use NumberDecimal instead of NumberDouble.

**Comment by kzangeli** [comment]
Old issue, but, I incremented the number of decimals to 9 (from 6).
So, please test again and close this issue if all good.
